text/0000-clarity-api-description.md

```diff
@@ -47,18 +41,8 @@ This type of transition, though, should have a URL and a method.
 
 We should add a section that defines what media types the server will respond with.
 
-## Add Schema to Resource
-
-A resource should be able to have its own schema, whereas now it cannot.
-
 # Unresolved questions
 
-## Category Element
-
-What is a "category"?
```

Comment:
This already seems clear in the [current specification](https://github.com/refractproject/refract-spec/blob/master/namespaces/api-namespace.md#category-element):

> ## Category (Element)

> Grouping element – a set of elements forming a logical unit of an API such as
> group of related resources or data structures.
> 
> A category element MAY include additional classification of the category.
> The classification MAY hint what is the content or semantics of the category.
> The classification MAY be extended and MAY contain more than one classes.
> 
> For example a `category` element may be classified both as `resourceGroup` and
`dataStructures` to denote it includes both resource and data structures.

/cc @smizell Is there any point I'm missing that's not documented correctly?